_posts/2017-01-26-create-an-fsharp-project-in-vscode.md

```diff
@@ -0,0 +1,166 @@
+---
+layout: post
+asset-type: post
+name: create-an-fsharp-project-in-vscode
+title: Create an F# project in VSCode
+date: 2017-01-26 10:00:00 +00:00
+author: Pedro Santos
+image:
+    src: /assets/img/custom/blog/2017-01-26-create-an-fsharp-project-in-vscode.jpg
+    attribution:
+      text: directions by Beat Tschanz
+      href: https://www.flickr.com/photos/btschanz/13793872333/
+tags:
+- F#
+- VSCode
+- Ionide 
+- Tutorial
+
+---
+
+## What we need:
+* Operating system  
+  * Windows
+  * MacOs
+  * Linux
+* CLR
+  * [.Net Framework](https://www.microsoft.com/net/download/framework) (Windows)
+  * [Mono](http://www.mono-project.com/]) (MacOs, Linux)
+* VSCode (https://code.visualstudio.com/)
+   * With ionide extensions for VSCode (http://ionide.io/)
+      * Ionide-FSharp
+      * Ionide-FAKE
+      * Ionide-Paket
+
+## How to step by step  
+1. Create new directory
+2. Move to new directory
+3. Type on console "Code ." or open VSCode and then open the new directory you just created
```

Comment:
you can enclose code snippets and keyboard combinations/shortcuts with Backticks (`) to improve readability